migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -251,6 +257,60 @@ func migrateRoles(gormDB *gorm.DB, postgresDB *pgxpool.Pool, legacyStore legacyr
 	return nil
 }
 
+func getReportConfigurationScopeID(reportConfiguration *storage.ReportConfiguration) (string, error) {
+	reportConfigurationScopeID := strings.TrimPrefix(reportConfiguration.GetScopeId(), accessScopeIDPrefix)
+	if replacement, found := accessScopeIDMapping[reportConfigurationScopeID]; found {
+		reportConfigurationScopeID = replacement
+	}
+	_, accessIDParseErr := uuid.FromString(reportConfigurationScopeID)
+	if accessIDParseErr != nil {
+		log.WriteToStderrf("failed to convert report configuration to postgres format, bad scope ID. Report Configuration ID: [%s] Name:[%s], error %v", reportConfiguration.GetId(), reportConfiguration.GetName(), accessIDParseErr)
+		return "", accessIDParseErr
+	}
+	return reportConfigurationScopeID, nil
+}
+
+// This function was added between 3.73 and 3.74 in order to cover an overlooked reference to access scope IDs.
+// Users who migrated to postgres with 3.73 may have report configurations relying on incorrect scope IDs.
+func migrateReportConfigurationScopeIDs(postgresDB *pgxpool.Pool) error {
+	ctx := sac.WithAllAccess(context.Background())
+	store := pgReportConfigurationStore.New(postgresDB)
+	var reportConfigs []*storage.ReportConfiguration
+	err := walkReportConfigurations(ctx, store, func(obj *storage.ReportConfiguration) error {
+		reportConfigurationScopeID, conversionErr := getReportConfigurationScopeID(obj)
+		if conversionErr != nil {
+			// The goal here is to fail open, and to log reports for which the report configuration update failed.
+			// The case where the scope ID update failed is logged in the called function.
+			return nil
+		}
+		obj.ScopeId = reportConfigurationScopeID
+		reportConfigs = append(reportConfigs, obj)
+		if len(reportConfigs) == batchSize {
+			err := store.UpsertMany(ctx, reportConfigs)
+			if err != nil {
+				batchIDs := make([]string, 0, len(reportConfigs))
+				for _, reportConfig := range reportConfigs {
+					batchIDs = append(batchIDs, reportConfig.GetId())
+				}
+				log.WriteToStderrf("failed to persist report configurations IDs [%s] to store %v", strings.Join(batchIDs, " "), err)
+				return err
```

Comment:
Since other places have changed (or suggested to be changed) to just logging error and returning nil and continue to migrate the rest of the configs, we can do the same here as well.